backend/JavaWebService/src/main/java/com/kainos/ea/resources/JobSpecificationService.java

```diff
@@ -0,0 +1,15 @@
+package com.kainos.ea.resources;
+
+import com.kainos.ea.db.RoleMapper;
+
+public class JobSpecificationService {
```

Comment:
don't need to separate job spec into it's own service - should use the role service instead